_posts/2014-04-02-using-modules-and-the-resolver.md

```diff
@@ -137,7 +137,7 @@ export default function(value, options) {
 import Ember from "ember";
 import trimHelper from './helpers/trim';
 
-Ember.Handlebars.registerBoundHelper('trim', trimHelper);
+Ember.Helper.helper('trim', trimHelper);
```

Comment:
This example should be removed, and the `Resolving Handlebars helpers` section should be reworded to make it clear that all helpers will be auto-resolved (regardless if they contain a dash or not).